src/mdast/tokensToMyst.ts

```diff
@@ -456,6 +489,16 @@ function hoistSingleImagesOutofParagraphs(tree: Root) {
   });
 }
 
+function nestSingleImagesIntoParagraphs(tree: Root) {
```

Comment:
This turns
```
type: root
children:
  - type: image
    ...
```
into
```
type: root
children:
  - type: paragraph
    children:
      - type: image
        ...
```

It's not super useful, but allows for some roundtripping to make commonmark tests pass.